fhirstarter/tests/test_interactions.py

```diff
@@ -25,32 +32,49 @@
 )
 
 
-def test_create(create_response_fixture: Response) -> None:
+@pytest.fixture(scope="module")
+def client(async_endpoints: bool) -> TestClient:
+    """Return a module-scoped test client with all interactions enabled."""
+    return create_test_client(
+        interactions=("create", "read", "search-type", "update"),
+        async_endpoints=async_endpoints,
+    )
```

Comment:
I'm overriding the `client` fixture in `conftest.py` here because I need it be module-scoped.